Fixed block rotation on chest, signs, and furnaces

OfficialKris opened the pull request:
<!-- Empty or bad Descriptions are not welcome, Don't waste my time -->

## Description
Fixes rotation on chest, signs, and furnaces. Block properties are not currently saved in world file.

## Testing
Open world two players
Place sign on ground or on wall, text faces you
Place chest, faces you
Place furnace, faces you

## Limitations
Sign on ground does allow for angles.

Please follow our [Coding Guidelines](https://github.com/Pumpkin-MC/Pumpkin/blob/master/CONTRIBUTING.md#coding-guidelines)

4lve commented:
I rewrote the whole property system, could you commit this into my repo instead or hold off until mine is merged to main? https://github.com/4lve/Pumpkin/tree/master

OfficialKris commented:
> I rewrote the whole property system, could you commit this into my repo instead or hold off until mine is merged to main? https://github.com/4lve/Pumpkin/tree/master

Yes, I reworked all of the changes to your new system. Works great

4lve commented:
> > I rewrote the whole property system, could you commit this into my repo instead or hold off until mine is merged to main? https://github.com/4lve/Pumpkin/tree/master
> 
> 
> 
> Yes, I reworked all of the changes to your new system. Works great

Nice, looks good

Snowiiii commented:
@OfficialKris any progress on this?

Snowiiii commented:
feel free to reopen
